test/verify/machineslib.py

```diff
@@ -1162,8 +1162,32 @@ def installWithErrorTest(dialog):
                                          start_vm=True))
         # End of tests for import existing disk as installation option
 
-        # test PXE Source
         if self.provider == "libvirt-dbus":
+            self.machine.execute("qemu-img create -f qcow2 /var/lib/libvirt/vmTestCustomDestination.qcow2 128M")
+            self.machine.execute("virsh pool-refresh libvirt")
+
+            # Check choosing existing volume as destination storage
+            createTest(TestMachines.VmDialog(self, sourceType='file',
+                                             location=config.NOVELL_MOCKUP_ISO_PATH,
+                                             memory_size=50, memory_size_unit='MiB',
+                                             storage_size=0, storage_size_unit='MiB',
```

Comment:
We don't choose size when choosing specific volume this this should be removed as well.